addons/note/static/src/models/activity_menu_view.js

```diff
@@ -62,10 +62,10 @@ addRecordMethods('ActivityMenuView', {
 
 patchFields('ActivityMenuView', {
     activityGroups: {
-        sort() {
+        sort(sortingRules) {
             return [
                 ['truthy-first', 'isNote'],
-                ...this._super(),
+                ...sortingRules,
```

Comment:
I would keep `this._super()`, for consistency of override in whole code

Reply:
It would be misleading in my opinion, as the original sort is not a function. Just so you know, I based this way of doing things on what I remember of how we used to patch identifying fields when there were still defined as an array.

Reply:
What's annoying with current approach is that you need to use the parameter (and also use a name).
Makes it both inconsistent with other patch ways that use `this._super`, but it's also very prone to mistake of forgetting using it (and instead maybe do `this._super`).

We don't care whether original is a function or not. `this._super` is just an easy way to call original (or combine, if there are other patches).